client/output.go

```diff
@@ -35,16 +34,6 @@ func (c *Client) GetOutput(ctx context.Context, buildID string, w io.Writer) err
 
 	dialer := websocket.DefaultDialer
 
-	// Due to this issue (https://github.com/gorilla/websocket/issues/601), it is not possible
```

Comment:
I'm a bit confused by this removal. Does this mean that the websocket dialer can never be passed a TLSClientConfig from Singularity (via transport rather than client) now?

Reply:
Good catch! I have reworked the code and added a unit test to validate the same. Also tested in Singularity `main` branch.